reviewkitteh/README.md

```diff
@@ -0,0 +1,80 @@
+# GitHub Pull Request to Slack Meow Workflow
```

Comment:
What is the Meow? I mean I understand, but in the title it looks like a mistake

Reply:
Done